app/helpers/channels_helper.rb

```diff
@@ -53,7 +53,7 @@ def channel_next_step_path(channel)
   end
 
   def channel_verification_status(channel)
-    if channel.verified?
+    if channel.verified? && channel.publisher.uphold_connection.is_member?
```

Comment:
I don't think we should add this, because having or not having verification with Uphold is independent of channel getting verified. Plus we should inform support team of this change

Reply:
Or I guess we should indicate elsewhere on a second field whether or not the channel connected to uphold.

Reply:
Yeah, this was just for a UI representation but I think we'll do something different on the homepage. 
https://github.com/brave-intl/publishers/pull/2094/commits/1addcb99452d89472851a87dd6fa11bef67617fd